Unix/buildtool

```diff
@@ -706,7 +714,7 @@ if [ "$arg1" = "cflags" -o "$arg1" = "cxxflags" ]; then
     r=""
 
     case "$platform" in
-        LINUX_IX86_GNU|LINUX_X86_64_GNU|MONTAVISTA_IX86_GNU|NETBSD_IX86_GNU)
+        LINUX_*|MONTAVISTA_IX86_GNU|NETBSD_IX86_GNU)
```

Comment:
So ditto for lines 717, 891, 1015, 1099, 1143, and a bunch of others. I won't mention the others, you can find them.